usagov/vote-gov-drupal
# VOTE-1521: link validator test update

**_Delete all details that do not apply to this PR!_**

## Jira ticket (required)
[Vote-1521](https://bixal-projects.atlassian.net/browse/VOTE-1521)

## Description (optional)
This Pr will update both Link validation tests, External (English and Spanish) and internal links.  Before we were just hitting the home page to migrate the test over.  Now that we have state pages and translations we need to update the test to do the end goal of validating the links 

> **NOTE**
> there are links that are failing, this will be address in another ticket the goal of this Pr is to just to update the 
> tests to run how they need to be in order to test the way we need them too. 

## Deployment and testing (required, if applicable)
### Testing steps
1. Pull down branch 
2. start local 
3. cd into `testing`
4. run `npm run cy:proofer` and `npm run cy:links` and verify that the tests are running (note failing links will be addressed and ticketed out for further investigation for a path forward)
